common/src/TrenchBroomApp.cpp

```diff
@@ -112,6 +112,37 @@ namespace TrenchBroom {
                 return;
             }
 
+            // initialize stylesheets
+            QString styleSheetString;
+
+            // load base stylesheet
+            QFile baseStyle = QFile(IO::pathAsQString(IO::SystemPaths::appDirectory() + IO::Path("styles/base.qss")));
+            assert(baseStyle.exists());
+            baseStyle.open(QFile::ReadOnly | QFile::Text);
+            QTextStream baseStyleStream(&baseStyle);
```

Comment:
Use a temporary instead of a variable, i.e. `styleSheetString += QTextStream(&baseStyle).readAll()...`